yarn.lock

```diff
@@ -1389,6 +1389,20 @@
   dependencies:
     postcss-value-parser "^4.2.0"
 
+"@dialectlabs/react-ui@^0.5.2":
```

Comment:
I think that we might've lost a version update somewhere, please check and remove this entry, since we should be using local versions instead. Thanks!

Reply:
this is really weird, when I run yarn in root it auto updates yarn.lock to this. anyways I removed it for now